extensions/patterns/configure_ec2/template_rhdh/create_ec2_instance.yml

```diff
@@ -0,0 +1,185 @@
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: cloud-configure-ec2
+  title: AWS Operations / Create EC2 Instance
+  description: This wizard will guide you on how to create EC2 instance in the Amazon cloud
+  namespace: default
+  tags:
+    - aap-operations
+    - intermediate
+    - cloud-aws-ops
+    - configure-ec2-pattern
+    - create-ec2-instance
+spec:
+  type: service
+  parameters:
+    - title: Prompts
+      description: Create EC2 instance in the Amazon cloud
+      required:
+        - token
+        - inventory
+      properties:
+        token:
+          title: Token
+          type: string
+          description: Oauth2 token
+          ui:field: AAPTokenField
+          ui:widget: password
+          ui:backstage:
+            review:
+              show: false
+        inventory:
+          title: Inventory
+          description: Please enter the inventory you want to use the services on
+          resource: inventories
+          ui:field: AAPResourcePicker
+        credentials:
+          title: Credentials
+          description: Select credentials for accessing the nodes this job will be ran
```

Comment:
Is this where they will select an existing AWS credential from AAP to use for the instance creation job? If so we might want to specify that this needs to be a credential of type "Amazon Web Services", and we can probably remove the SSH credential info since that isn't relevant for this use case.

Reply:
done